### DISCUSSION TITLE
phpMyAdmin versions switch like PHP & Apache

### DISCUSSION THREAD

**Discussion by giorgosstab** [leokhoa/laragon#670] [Ideas]
Think the possibility that you need load an old project at PHP 5.4 or 5.6 and you have installed latest version of phpMyAdmin. Then you cannot open phpMyAdmin because PHP is not compatible with version that need.

#443 That is the only open discussion for that!

**Comment by RobiNN1** [comment]
Download pma4 (`C:\laragon\etc\apps` folder) and create alias.

In `C:\laragon\etc\apache2\alias` is file `phpmyadmin.conf` copy it to `phpmyadmin4.conf` and change path to the pma4.

Example

```conf
Alias /phpmyadmin4 "C:/laragon/etc/apps/phpMyAdmin4/"

# to give access from outside
# replace the lines
#
# Require local
#
# by
#
# Require all granted
#

<Directory "C:/laragon/etc/apps/phpMyAdmin4/">
	Options Indexes FollowSymLinks MultiViews
	AllowOverride all

	Require local
</Directory>

```

After server restart you should have available `localhost/phpmyadmin` and `localhost/phpmyadmin4`.

**Reply by giorgosstab** [reply]
Dude I have done this before you. I am wrb developer I have many extra custom apps in Paragon. First of all that is time losing for me and not easy. With other alias an think about that I will need to add 10x different versions of phpmyadmin and need to remember each time what is the alias of each one.

The same think it's when I want to add other php version but the author decided to make it easy for user with right click and change version. I open that discussion as a feature opportunity to be on list for next versions. Thanks for the fast answer BTW.

**Reply by RobiNN1** [reply]
At least it's there for others, not everyone knows about it.

**Reply by giorgosstab** [reply]
I aggre with you. Think that if some1 use laragon it's a web developer. If he give in app a bit time he will understand the logic. It's easy app and expendable if want add extra features.

**Comment by giorgosstab** [comment]
BTW I will use your app for cache. Nice app mate! @RobiNN1

**Reply by RobiNN1** [reply]
Thanks. There is also open issue https://github.com/leokhoa/laragon/issues/313 I would like it as the default because the current bundled solutions are bad and really ugly.

**Reply by giorgosstab** [reply]
Nice I will w8 that feature because I am using your app as alias and remove the ugly old memcache and redis of laragon.
